packages/expo-sqlite/android/src/main/java/expo/modules/sqlite/SQLiteModule.kt

```diff
@@ -83,7 +83,7 @@ class SQLiteModule : Module() {
           if (bindArgs[i - 1] == null) {
             statement.bindNull(i)
           } else {
-            statement.bindString(i, bindArgs[i - 1])
+            statement.bindString(i, bindArgs[i - 1]!!)
```

Comment:
nit: maybe `let arg = bindArgs[i - 1]` and do small refactor than using the `!!` here